lamindb_setup/_connect_instance.py

```diff
@@ -196,6 +191,11 @@ def connect(
         storage: Load the instance with an updated default storage.
     """
     isettings: InstanceSettings = None  # type: ignore
+
+    kwargs.get("access_token", None)
+    _raise_not_found_error = kwargs.get("_raise_not_found_error", True)
+    _test = kwargs.get("_test", False)
```

Comment:
@falexwolf are you fine with moving all internal arguments to `kwargs` in `connect` and `init`?

Reply:
Ok!

Reply:
I thought about this again and what I'd prefer is to have a properly typed signature of a function called `connect_internal()` that's wrapped by the user-facing `connect()`.

This way we eliminate **kwargs. We really shouldn't have them anywhere unless those places where we can't come up with a different solution.

Reply:
The user-facing `connect()` will be super minimal and simple after also incorporating this change:  https://github.com/laminlabs/lamindb-setup/issues/862

Reply:
And then we'll also be ready to attack this: https://github.com/laminlabs/pfizer-lamin-usage/issues/25

Reply:
I agree, i just think that this probably to do in another PR, it requires a lot of other things, its own tests etc that i don't want to do here.

Reply:
Ok!